src/applications/vaos/tests/list/past-appointments.unit.spec.jsx

```diff
@@ -15,23 +19,199 @@ const initialState = {
   },
 };
 
+const pastDate = moment().subtract(3, 'days');
+
 describe('VAOS integration: past appointments', () => {
+  it('should show select date range dropdown', async () => {
```

Comment:
Could we write a test that checks that changing the dropdown value and updating fetches a new range of appointments?